msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignInEmailPasswordTest.kt

```diff
@@ -30,26 +30,49 @@ import kotlinx.coroutines.test.runTest
 import org.junit.Assert
 import org.junit.Test
 
-class SignInTest : NativeAuthPublicClientApplicationAbstractTest() {
+class SignInEmailPasswordTest : NativeAuthPublicClientApplicationAbstractTest() {
 
     override val configType = ConfigType.SIGN_IN_PASSWORD
 
+    /**
+     * Use valid email and password to get token.
+     * (hero scenario 15, use case 1.2.1, Test case 37)
+     */
     @Test
-    fun testSignInErrorSimple() = runTest {
+    fun testSuccess() = runTest {
         val username = config.email
```

Comment:
The email and the password are needed in every test, maybe it's better to make these variables members of the class and initialize them only once and not need to declare them in every test.